### PR TITLE
Upgrade syft version

### PR DESCRIPTION
**Pull request by mkosiarc**
[STONEBLD-1491](https://issues.redhat.com//browse/STONEBLD-1491)

### CODE REVIEW

File: task/sbom-json-check/0.1/sbom-json-check.yaml [hunk 1]
```diff
@@ -18,7 +18,7 @@ spec:
       name: TEST_OUTPUT
   steps:
   - name: sbom-json-check
-    image: quay.io/redhat-appstudio/hacbs-test:v1.1.0@sha256:82b43bffe4eacc717239424f64478b18f36528df47c2d11df3a8d031e81a3c67
+    image: quay.io/mkosiarc_rhtap/test-hacbs:test2
```
Comment (mkosiarc) [inline task/sbom-json-check/0.1/sbom-json-check.yaml:21]: I'll change this, waiting for associated PR https://github.com/redhat-appstudio/hacbs-test/pull/117

Comment (mkosiarc) [reply task/sbom-json-check/0.1/sbom-json-check.yaml:21]: PR merged, waiting for release of a new image

**Comment by mkosiarc** [comment]
Image was released, ready for review.

File: task/sbom-json-check/0.1/sbom-json-check.yaml [hunk 2]
```diff
@@ -18,7 +18,7 @@ spec:
       name: TEST_OUTPUT
   steps:
   - name: sbom-json-check
-    image: quay.io/redhat-appstudio/hacbs-test:v1.1.0@sha256:82b43bffe4eacc717239424f64478b18f36528df47c2d11df3a8d031e81a3c67
+    image: quay.io/redhat-appstudio/hacbs-test:v1.1.1
```
Comment (MartinBasti) [inline task/sbom-json-check/0.1/sbom-json-check.yaml:21]: can you keep digest there?

Comment (mkosiarc) [reply task/sbom-json-check/0.1/sbom-json-check.yaml]: rebased pr, no longer needed.

**Review by Michkov — APPROVED** [review]

**Pull request merged**